keyboards/handwired/technicpad/info.json

```diff
@@ -0,0 +1,42 @@
+{
+    "manufacturer": "nwhirschfeld",
+    "keyboard_name": "TechnicPad",
+    "maintainer": "nwhirschfeld",
+    "bootloader": "caterina",
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
```

Comment:
Duplicating defaults should be avoided
```suggestion
```

Reply:
solved in https://github.com/qmk/qmk_firmware/pull/23238/commits/808b271c2898a5b048c03d578a69c0c1fb882b6e